py_zipkin/logging_helper.py

```diff
@@ -332,8 +328,13 @@ def __init__(self, transport_handler, max_portion_size=None):
         self.transport_handler = transport_handler
         self.max_portion_size = max_portion_size or self.MAX_PORTION_SIZE
 
+        if getattr(self.transport_handler, 'get_max_payload_bytes', None):
+            self.max_payload_bytes = self.transport_handler.get_max_payload_bytes()
```

Comment:
I think it's worthwhile to create an interface/abstract type for the transport handler so that the API is a little more clearly defined. That way we can say `isinstance(ITransportHandler)` or whatever instead of duck-typing.

Reply:
good idea!